docs/integration/index.md

```diff
@@ -0,0 +1,90 @@
+# DTS Integration Guide
+
+This document lists all of the things you must do in order to integrate your
+database with the BER Data Transfer System (DTS) in order to take advantage of
+its file and metadata transfer capabilities.
```

Comment:
"all the things" -- could this be reworded as "things" isn't a very useful or descriptive word. "This document goes over the requirements to integrate..."?